src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -1096,29 +1097,35 @@ private boolean undeclaredWriteInsEnabled() {
   }
 
   enum Provider {
-    CDF("CDF"),
-    CLEAR_BALLOT("Clear Ballot"),
-    DOMINION("Dominion"),
-    ESS("ES&S"),
-    HART("Hart"),
-    PROVIDER_UNKNOWN("Provider unknown");
-
-    private final String label;
-
-    Provider(String label) {
-      this.label = label;
+    CDF("cdf", "CDF"),
+    CLEAR_BALLOT("clearBallot", "Clear Ballot"),
+    DOMINION("dominion", "Dominion"),
+    ESS("ess", "ES&S"),
+    HART("hart", "Hart"),
+    PROVIDER_UNKNOWN("providerUnknown", "Provider unknown");
+
+    private final String internalLabel;
+    private final String uiLabel;
+
+    Provider(String internalLabel, String uiLabel) {
+      this.internalLabel = internalLabel;
+      this.uiLabel = uiLabel;
```

Comment:
Nit: might be better to call these `guiLabel`, since the CLI is also technically a UI.

Reply:
that's fair